packages/components/Avatar/src/useAvatar.ts

```diff
@@ -18,6 +18,7 @@ import { titles, multiWordTitles } from './titles';
 export const useAvatar = (props: AvatarProps): AvatarInfo => {
   const {
     avatarColor,
+    accessible,
```

Comment:
We can use default parameters instead.
```js
accessible = true
```

Reply:
I'm going to leave this as is since most of our other props follow the current pattern.

(Also personally I'm not a huge fan of default props, since it makes it less obvious where the value of a variable is coming from when reading code, but that's a personal opinion).